site/docs/spark.md

```diff
@@ -136,6 +136,11 @@ To show table history, run:
 ```scala
 spark.read.format("iceberg").load("db.table.history").show(truncate = false)
 ```
+
+To show table history by HDFS path, run:
+```scala
+spark.read.format("iceberg").load(table.location() + "#" + MetadataTableType.HISTORY).show( truncate = false)
```

Comment:
Where does `table` come from? And why require loading `MetadataTableType`?

I think it would be better to use just a string, following the pattern of the other HDFS examples:

```scala
spark.read.format("iceberg").load("hdfs://nn:8020/warehouse/path/db.db/table#history").show(truncate = false)
```

Reply:
Thank you for your advice! I have finished the modification.